pipeline/outputs/dynatrace.md

```diff
@@ -0,0 +1,61 @@
+| json_date_format           | Date format standard for JSON                                                                                                                                                                                                       | iso8601                                            |
+| json_date_key              | Fieldname specifying message timestamp                                                                                                                                                                                              | timestamp                                          |
+| tls                        | Specify to use TLS                                                                                                                                                                                                                  | on                                                 |
+| tls.verify                 | TLS verification                                                                                                                                                                                                                    | on                                                 |
+
+## Getting Started
+
+1. Get a [Dynatrace API](https://docs.dynatrace.com/docs/shortlink/api-authentication) token with the `logs.ingest` (Ingest Logs) scope.
+2. Determine your Dynatrace [environment ID](https://docs.dynatrace.com/docs/shortlink/monitoring-environment#environment-id).
+3. In your main Fluent Bit configuration file, append the following _Output_ section:
+
+```text
+[OUTPUT]
+    name         http
+    match        *
+    header       Content-Type application/json; charset=utf-8
+    header       Authorization Api-Token {your-API-token-here}
+    allow_duplicated_headers false
+    host         {your-environment-id}.live.dynatrace.com
+    Port         443
+    URI          /api/v2/logs/ingest
+    Format       json
+    json_date_format iso8601
+    json_date_key timestamp
+    tls          On
+    tls.verify   On
+```
+
+4. Once the configuration is applied and new logs are appended, verify their receipt within Dynatrace. This can be done using Dynatrace [Notebooks](https://docs.dynatrace.com/docs/observe-and-explore/dashboards-and-notebooks/notebooks) with a Dynatrace Query Language query, for example:
+
+    ```
+    fetch logs
+    ```
+
+## References
+
+* [Dynatrace Fluent Bit documentation](https://docs.dynatrace.com/docs/shortlink/lma-stream-logs-with-fluent-bit)
+* [Fluent Bit integration in Dynatrace Hub](https://www.dynatrace.com/hub/detail/fluent-bit/?filter=log-management-and-analytics) 
+* [Video: Stream a Log File to Dynatrace using Fluent Bit](https://www.youtube.com/watch?v=JJJNxhtJ6R0)
+* [Blog: Easily stream logs from Fluent Bit to Dynatrace](https://www.dynatrace.com/news/blog/easily-stream-logs-with-fluent-bit-to-dynatrace/)
```

Comment:
For clarity, we encourage the use of hyphens for bullets instead of asterisks, as it helps avoid Markdown scanning confusion with bold codes.

```suggestion
- [Dynatrace Fluent Bit documentation](https://docs.dynatrace.com/docs/shortlink/lma-stream-logs-with-fluent-bit)
- [Fluent Bit integration in Dynatrace Hub](https://www.dynatrace.com/hub/detail/fluent-bit/?filter=log-management-and-analytics) 
- [Video: Stream a Log File to Dynatrace using Fluent Bit](https://www.youtube.com/watch?v=JJJNxhtJ6R0)
- [Blog: Easily stream logs from Fluent Bit to Dynatrace](https://www.dynatrace.com/news/blog/easily-stream-logs-with-fluent-bit-to-dynatrace/)
```